### ISSUE TITLE
How to allow a specific website?

### ISSUE THREAD

**Issue by impala75** [mozilla/contain-facebook#531]
Hello,

Sorry, I am not sure it is the right place to ask this.

I would like to know how to allow a specific website, that is, to add it to the container.

For example, there is a blog I like visiting, which embeds Instagram pictures. These pictures are blocked by Facebook Container. How to add this website to the container so that I can see the pictures?

**Comment by maxxcrawford** [comment]
This feature is planned for our next major release. See that issues' progress on #422 

Duplicate of #422

**Comment by JeremyMrtn** [comment]
Waiting for #422 issue, is there a manual way to allow specific websites ?
Thanks for the work already done.
